### PR TITLE
fix: prevent multiple clicks

### PR DESCRIPTION
**Pull request by pauldambra**
## Problem

I tried to select the text in a template description and ended up creating three copies of the dashboard

## Changes

set state so that only the first click can be acted on

## How did you test this code?

👀

### CODE REVIEW

File: frontend/src/scenes/dashboard/newDashboardLogic.ts [hunk 1]
```diff
@@ -160,6 +173,7 @@ export const newDashboardLogic = kea<newDashboardLogicType>([
                     lemonToast.error(`Could not create dashboard: ${message}`)
                 }
             }
+            actions.setIsLoading(false)
```
Comment (yakkomajuri) [inline frontend/src/scenes/dashboard/newDashboardLogic.ts:176]: when is this set to `true` in this context?

**Comment by lharries** [comment]
Do we want to have the whole modal turn into a loading spinner after the first click?

**Review by yakkomajuri — APPROVED** [review]
small nit that IIRC `chore` should be for changes only on files that don't affect the app's source code i.e. tests, package.json etc

File: frontend/src/scenes/dashboard/newDashboardLogic.ts (hunk 1 above)
Comment (pauldambra) [reply frontend/src/scenes/dashboard/newDashboardLogic.ts:176]: It's set to true in the click handler

File: frontend/src/scenes/dashboard/newDashboardLogic.ts [hunk 2]
```diff
@@ -72,6 +73,16 @@ export const newDashboardLogic = kea<newDashboardLogicType>([
         }),
     }),
     reducers({
+        isLoading: [
+            false,
+            {
+                setIsLoading: (_, { isLoading }) => isLoading,
+                hideNewDashboardModal: () => false,
+                submitNewDashboard: () => true,
```
Comment (pauldambra) [inline frontend/src/scenes/dashboard/newDashboardLogic.ts:81]:
```suggestion
```

**Pull request merged**